ert_shared/storage/blob_api.py

```diff
@@ -44,4 +44,8 @@ def add_blob(self, data):
         return data_frame
 
     def get_blob(self, id):
-        return self._session.query(ErtBlob).get(id)
+        if not isinstance(id, list):
+            id = [id]
+
+        return self._session.query(ErtBlob).filter(ErtBlob.id.in_(id)).yield_per(100).enable_eagerloads(False)
```

Comment:
The yield_per argument doesn't seem to affect things much in my experiments. Probably would make an impact if data was huge though.

Reply:
Please let `id` be singular and add `get_blobs`? Not sure how `filter` compares to `get`.